src/NuGet.Core/NuGet.Configuration/Settings/Items/AuthorItem.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using NuGet.Shared;
+
+namespace NuGet.Configuration
+{
+    public sealed class AuthorItem : TrustedSignerItem, IEquatable<AuthorItem>
+    {
+        public override string ElementName => ConfigurationConstants.Author;
+
+        protected override HashSet<string> RequiredAttributes => new HashSet<string>() { ConfigurationConstants.NameAttribute };
+
+        public AuthorItem(string name, params CertificateItem[] certificates)
+            : base(name, certificates)
+        {
+        }
+
+        internal AuthorItem(XElement element, SettingsFile origin)
+            : base(element, origin)
+        {
+        }
+
+        internal override SettingBase Clone()
```

Comment:
Return type should be `AuthorItem`

Reply:
This is an override from `SettingBase` the type cannot be overwritten.

Reply:
You can have a generic class that SettingsBase and the rest implement. 
That way the return type of clone can be the type of the class itself. 

We already have ICloneable too, but I don't like ICloneable cause it doesn't enforce any contracts.